src/components/tables/EditableTranscriptsTable.tsx

```diff
@@ -1,216 +0,0 @@
-import config from "@/config/config.json";
-import { useUserReviews } from "@/services/api/reviews";
-import { useClaimTranscript, useTranscripts } from "@/services/api/transcripts";
-import { calculateReadingTime, isReviewActive, isReviewPending } from "@/utils";
```

Comment:
`isReviewActive` & `isReviewPending` are only used in this file. We are removing this one so we should remove those also.